README.md

```diff
@@ -6,10 +6,20 @@ Requires users to accept terms of service before accessing data. Text and langua
 
 ## 🔗 Display on public shares
 
-The setting applies to shares via link or mail (with and without password protection). Default is off `0`
+The setting applies to shares via link or mail (with and without password protection).
+
+Default is disabled: `0`
 ```
 ./occ config:app:set terms_of_service tos_on_public_shares --value '1'
 ```
+## 👤 Exclude logged in users
+
+To exclude logged in users from the terms of service, set this config to `0`.
```

Comment:
```suggestion
To exclude registered users from accepting the terms of service, set this config to `0`.
Therefore, only email- and public link sharees have to accept the terms of service.
```

Reply:
Adjusted the headline as well to "registered"

Reply:
> Adjusted the headline as well to "registered"

👍
makes the setting terminal clear